### PR TITLE
Update the display name of MSATP connector to Microsoft Defender for Endpoint

### PR DESCRIPTION
**Pull request by mdazam1942**

### CODE REVIEW

File: stix_shifter_modules/msatp/configuration/lang_en.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         },
         "selfSignedCert": {
             "label": "Microsoft Defender ATP Certificate",
```
Comment (delliott90) [inline stix_shifter_modules/msatp/configuration/lang_en.json:17]: The label should also be updated.

File: stix_shifter_modules/msatp/stix_translation/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# MSATP UDS Connector
+# Microsoft Defender for Endpoint(previously Microsoft Defender ATP) UDS Connector
```
Comment (delliott90) [inline stix_shifter_modules/msatp/stix_translation/README.md:1]: Add a space at the start of the opening parenthesis.

File: stix_shifter_modules/msatp/configuration/lang_en.json (hunk 1 above)
Comment (mdazam1942) [reply stix_shifter_modules/msatp/configuration/lang_en.json]: done

File: stix_shifter_modules/msatp/stix_translation/README.md (hunk 2 above)
Comment (mdazam1942) [reply stix_shifter_modules/msatp/stix_translation/README.md]: done

**Review by delliott90 — APPROVED** [review]

**Pull request merged**